src/graphs.rs

```diff
@@ -46,30 +46,27 @@ struct LockfileNpmGraphPackage {
   dependencies: BTreeMap<String, LockfileNpmPackageId>,
 }
 
-#[derive(Debug, Default)]
+#[derive(Debug)]
 struct LockfileJsrGraphPackage {
   /// Root ids that transitively reference this package.
   root_ids: HashSet<LockfilePkgId>,
+  integrity: String,
   dependencies: BTreeSet<LockfilePkgReq>,
 }
 
 /// Graph used to analyze a lockfile to determine which packages
 /// and remotes can be removed based on config file changes.
-pub struct LockfilePackageGraph<FNvToJsrUrl: Fn(&str) -> Option<String>> {
+pub struct LockfilePackageGraph {
   root_packages: HashMap<LockfilePkgReq, LockfilePkgId>,
   packages: HashMap<LockfilePkgId, LockfileGraphPackage>,
   remotes: BTreeMap<String, String>,
-  nv_to_jsr_url: FNvToJsrUrl,
```

Comment:
Don't need this anymore because we don't store jsr urls in the "remotes" section anymore.